pyproject.toml

```diff
@@ -9,6 +9,7 @@ readme = "README.md"
 python = "^3.10"
 Django = "^4.2.1"
 gunicorn = "^20.1.0"
+black = "^23.3.0"
```

Comment:
Can you add this as a `dev` dependency?

Reply:
Use `poetry remove black && poetry add -D black`.

Reply:
done